theme/theme.inc

```diff
@@ -234,11 +234,19 @@ function islandora_solr_metadata_query_fields($object, &$solr_fields) {
               // Will get stripped out later.
               return;
             }
-            $solr_query = format_string('!field:"!value"', array(
+            $solr_query = format_string('!field:!value', array(
               '!field' => $solr_field,
-              '!value' => urlencode(islandora_solr_replace_slashes(islandora_solr_lesser_escape($original_value))),
+              '!value' => islandora_solr_facet_escape($original_value),
+            ));
+            return t('<a href="@url">@val</a>', array(
```

Comment:
Because that seemed to take me back to the original problem of double encoding of some characters.